docs/croissant-spec.md

```diff
@@ -263,14 +263,40 @@ corresponding properties must be defined using `subField`.
 
 A field may have more than a single assigned `dataType`, in which case at least
 one must inform about the expected type of data (eg: `sc:Text`), while other
-types inform about the semantic being used.
+types inform about the semantic being used, possibly semantics with ML meaning.
 
 **range**:	[sc:Text](https://schema.org/Text), [sc:URL](https://schema.org/URL)
 
 **domain**:	[Field](#field)
 
+[🔗](#known-supported-data-types) Known supported data types:
 
-In the following example, the `url` field is expected to be a URL , which
+| `dataType` | Usage |
+| -------- | ----- |
+| [**`sc:Boolean`**](https://schema.org/Boolean) | Describes a boolean. |
+| [**`sc:Date`**](https://schema.org/Date) | Describes a date. |
+| [**`sc:Float`**](https://schema.org/Float) | Describes a float. |
+| [**`sc:Integer`**](https://schema.org/Integer) | Describes an integer. |
+| [**`sc:Text`**](https://schema.org/Text) | Describes a string. |
+| [**`sc:URL`**](https://schema.org/ImageObject) | Describes a URL. |
+| [**`sc:ImageObject`**](https://schema.org/ImageObject) | Describes a field containing the content of an image (pixels). |
+| [**`ml:BoundingBox`**](http://mlcommons.org/schema/BoundingBox) | Describes a bounding box. |
```

Comment:
If we define this under the ml namespace, then we also need to describe it as a class, right? 
We may also consider supporting https://schema.org/GeoShape, which has propperties for box, polygon, etc.

Reply:
There is also [dcat:bbox](https://www.w3.org/TR/vocab-dcat-3/#Property:location_bbox).

Reply:
Added class for ml:BoundingBox, with a format property, so updated COCO dataset accordingly.

Please let me know if that makes sense. It feels somehow akward to me to use the same "ml:" namespace for both:
1. classes, which we can use in `@type` property, and 
2. types we can specify in  `dataType`.

Not sure what `dataType` value should be if we have `"@type": "ml:BoundingBox"`.

I am not sure about using `sc:GeoShape` for bounding boxes. It seems very loosely defined to me... Ditto for `dcat:bbox`. But happy to be convinced otherwise.